tests/integration/test_notes.py

```diff
@@ -28,3 +29,19 @@ def test_it_returns_404_with_invalid_ticket(self, valid_header):
             headers=valid_header,
         )
         assert is_valid(response, 404)  # Bad Request- 'Invalid Ticket'
+
+    def test_it_deletes_one_note(
+        self, valid_header, create_note, create_admin_user, create_ticket
+    ):
+        note = create_note(create_admin_user(), create_ticket())
+
+        response = self.client.delete(
+            f"{self.endpoint}/{note.id}/notes", headers=valid_header
```

Comment:
I guess I didn't think this all of the way through, I'm sorry.  This is going to change the endpoint for the .post as well as the .delete, since the .POST is only accepting the ticket_id.  How do we feel about changing the endpoint in app.py to:

```python
    api.add_resource(Note, "tickets/<int:ticket_id>/notes", "tickets/<int:ticket_id>/notes/<int:note_id>")
```

and then the test code to: 

```python
            f"{self.endpoint}/{note.ticketid}/notes/{note.id}", headers=valid_header
```

as Anthony suggests.